src/anf-preview/setup.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python
+
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from codecs import open
+from setuptools import setup, find_packages
+
+VERSION = "0.0.1"
```

Comment:
If this is still in development, and the goal is to hold off on releasing, this is fine. If the goal is to release for preview soon, this will need to be bumped to 0.1.0.